egui_demo_lib/src/easy_mark/easy_mark_editor.rs

```diff
@@ -107,22 +104,43 @@ impl EasyMarkEditor {
             }
         }
     }
+    fn nested_hotkeys(ui: &mut egui::Ui) {
+        let _ = ui.label("CTRL+B *bold*");
+        let _ = ui.label("CTRL+N `code`");
+        let _ = ui.label("CTRL+I /italics/");
+        let _ = ui.label("CTRL+L $subscript$");
+        let _ = ui.label("CTRL+Y ^superscript^");
+        let _ = ui.label("ALT+SHIFT+Q ~strikethrough~");
+        let _ = ui.label("ALT+SHIFT+W _underline_");
+        let _ = ui.label("ALT+SHIFT+E two spaces");
```

Comment:
Wait I missed  this in my first review - what weird shortcut is this? O.o

Reply:
the two spaces is a placeholder for tab indenting